.github/ISSUE_TEMPLATE/moderator_application.md

```diff
@@ -2,7 +2,9 @@
 name: Community Moderator Request
 about: Request moderator priviledges on a Kubernetes property
 title: 'REQUEST: New moderator for <your-GH-handle> of <k8s property>'
-labels: area/community-management
+labels:
```

Comment:
I'm pretty sure this is a comma separated list instead of a yaml array

Reply:
Thought so too...but comma separated list looks like https://github.com/prowtest/prow-test-2/blob/master/.github/ISSUE_TEMPLATE/moderator_application.md i.e. the yaml frontmatter is not rendered as...well, yaml frontmatter.